sdk/playwrighttesting/microsoft-playwright-testing/src/utils/serviceClient.ts

```diff
@@ -144,11 +152,20 @@ export class ServiceClient {
     if (response.status === 200) {
       return JSON.parse(response.bodyAsText!) as StorageUri;
     } else {
-      throw new Error(`Received status ${response.status} from service from GET StorageUri call.`);
+      this.handleErrorResponse(response, 'getStorageUri');
     }
+    throw new Error(`Received status ${response.status} from service from GET StorageUri call.`);
   }
 
   private getServiceEndpoint(): string {
     return process.env["PLAYWRIGHT_SERVICE_REPORTING_URL"]!;
   }
+
+  private handleErrorResponse(response: PipelineResponse, action: string) {
+    if (response.status === 400 || response.status === 401 || response.status === 403) {
+      process.stdout.write(`\n${Constants.ERROR_MESSAGE[response.status].action}`);
+    } else {
```

Comment:
we can remove else as you are anyway throwing it in caller method

Reply:
yes